product_pricelist_auto_mailing/models/product_pricelist.py

```diff
@@ -10,14 +10,20 @@ class ProductPricelist(models.Model):
 
     mailable_user_ids = fields.Many2many('res.users', readonly=True, copy=False)
 
+    def send_mail(self, mailable_pricelist_ids):
+        body = []
+        ctx = self.env.context.copy()
+        for pricelist in mailable_pricelist_ids:
+            body.append(_('The pricelist %s, has changed for users: %s') % (pricelist.name, ' ,'.join(pricelist.mailable_user_ids.mapped('name'))))
```

Comment:
It's a good time to change this text by this one IMO:
```suggestion
            body.append(_('The pricelist %s has been changed by the following users: %s') % (pricelist.name, ' ,'.join(pricelist.mailable_user_ids.mapped('name'))))
```
Then please change it and update translation files. This changes can be placed along this commit, adding to the description that an email text improvement has been made